pkg/stack/options.go

```diff
@@ -30,9 +31,42 @@ import (
 
 var stack string
 
+func GetStacks() ([]string, error) {
+	stackFiles, err := utils.GlobInDir(".", "nitric-*.yaml")
+	if err != nil {
+		return nil, err
+	}
+
+	stacks := []string{}
+
+	for _, sf := range stackFiles {
+		stacks = append(stacks, strings.TrimSuffix(strings.TrimPrefix(sf, "nitric-"), ".yaml"))
+	}
+
+	return stacks, nil
+}
+
 // Assume the project is in the currentDirectory
 func ConfigFromOptions() (*Config, error) {
-	return configFromFile("nitric-" + stack + ".yaml")
+	sName := stack // Default to the supplied stack
+	if sName == "" {
+		stacks, err := GetStacks()
+		cobra.CheckErr(err)
```

Comment:
```
if err!=nil{
return err
}
```
we have a error that is getting returned (same below)

Reply:
Resolved, copy and paste error 🤦‍♂️